docs/operations/rbac-and-namespaces.md

```
@@ -0,0 +1,97 @@
+## Where jobs run
+
+Job pods run in `executor_config.namespace` when it is set, otherwise in the
+runner's own namespace.
+
+By default a runner may only target its **own** namespace. A Runner author
+picking an arbitrary namespace would otherwise have the operator bind their
+ServiceAccount, and run their jobs, in somewhere like `kube-system`.
+
+To permit specific build namespaces, start the operator with:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)